pkg/core/multi_search.go

```diff
@@ -30,31 +30,58 @@ import (
 	"github.com/zinclabs/zinc/pkg/uquery/timerange"
 )
 
+// isMatchIndex("abc", "a")  false
+// isMatchIndex("abc", "a*") true
+// isMatchIndex("abc", "*bc") true
+// isMatchIndex("abc", "bc") false
+// isMatchIndex("abc", "abc") true
+func isMatchIndex(zincIndexName, indexName string) bool {
+	// the old code: indexName == "" || (indexName != "" && strings.HasPrefix(zincIndexName, indexName[:len(indexName)-1]))
+
```

Comment:
I think we can delete it, i known your mean, but we don't need keep this in code for main branch.